### ISSUE TITLE
Different coordinate notation

### ISSUE THREAD

**Issue by rnmitchell** [MarcoDiSalvo90/RhoTermPredict#6]
After examining the output of RhoTermPredict, I am curious as to why it appears there are two different coordinate notations being used. The palindromic sequences (for example, ATAT) are reported as Nucleotides 55-58 and 63-66, indicating a 0-based closed interval notation. But earlier, the coordinates of the RUT site are listed as 1012-1090, suggesting they're using 0-based open interval notation, since the corresponding sequence is 78 nucleotides long. Was this deliberate? And if so, why? 

Thank you!

**Comment by MarcoDiSalvo90** [comment]
Yes, for the RUT site coordinates we used a 0-based open interval notation, while regarding the palindromic sequence coordinates, they are related to the 78-long region downstream the RUT site. So, this was the convention that we decided to use.

Thank you
